packages/dart/noports_core/lib/src/srv/srv.dart

```diff
@@ -52,17 +53,35 @@ abstract class Srv<T> {
   static Srv<SocketConnector> dart(
     String host,
     int streamingPort, {
-    required int localPort,
-    required bool bindLocalPort,
+    int? localPort,
+    bool? bindLocalPort,
     String? rvdAuthString,
     String? sessionAESKeyString,
     String? sessionIVString,
   }) {
     return SrvImplDart(
       host,
       streamingPort,
-      localPort: localPort,
-      bindLocalPort: bindLocalPort,
+      localPort: localPort!,
+      bindLocalPort: bindLocalPort!,
+      rvdAuthString: rvdAuthString,
+      sessionAESKeyString: sessionAESKeyString,
+      sessionIVString: sessionIVString,
+    );
+  }
+
+  static Srv<SSHSocket> inline(
```

Comment:
Added new `inline` method to create the "Inline" Srv which doesn't bind to a port (and thus is good for mobile app sandboxes etc)